src/main/java/com/fasterxml/jackson/databind/deser/impl/JDKValueInstantiators.java

```diff
@@ -23,19 +24,28 @@ public static ValueInstantiator findStdValueInstantiator(DeserializationConfig c
             Class<?> raw)
     {
         if (raw == JsonLocation.class) {
-            return new JsonLocationInstantiator();
+            return JsonLocationInstantiator.INSTANCE;
```

Comment:
I guess it is ok although I had left it on-demand on purpose, reducing amount of eager construction.